sdcm/remote/kubernetes_cmd_runner.py

```diff
@@ -61,16 +61,20 @@ def close_proc_stdin(self) -> None:
 
     def start(self, command: str, shell: str, env: dict) -> None:
         with self._ws_lock:
-            self.process = k8s.stream.stream(self._k8s_core_v1_api.connect_get_namespaced_pod_exec,
-                                             name=self.context.config.k8s_pod,
-                                             container=self.context.config.k8s_container,
-                                             namespace=self.context.config.k8s_namespace,
-                                             command=[shell, "-c", command],
-                                             stderr=True,
-                                             stdin=True,
-                                             stdout=True,
-                                             tty=False,
-                                             _preload_content=False)
+            try:
+                self.process = k8s.stream.stream(
+                    self._k8s_core_v1_api.connect_get_namespaced_pod_exec,
+                    name=self.context.config.k8s_pod,
+                    container=self.context.config.k8s_container,
+                    namespace=self.context.config.k8s_namespace,
+                    command=[shell, "-c", command],
+                    stderr=True,
+                    stdin=True,
+                    stdout=True,
+                    tty=False,
+                    _preload_content=False)
+            except k8s.client.rest.ApiException as exc:
+                raise ConnectionError(str(exc))
```

Comment:
Please, change it to `raise ... from None`

Reply:
Done